### ISSUE TITLE
Quest mobs should respawn if item not collected

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#108]
```
What steps will reproduce the problem?
1. Kill a quest mob and another in the same fight
2. The 2nd mobs loot is collected if it is killed after the quest mob
3.

What is the expected output? What do you see instead?
Expected: Both mobs loot should be collected, or the quest mob should re-spawn 
at a later visit to the area if you didn't collect the loot item.

Instead: You get stuck without the quest item and screwed into re-starting 
since the quest mob doesn't re-spawn : \ (I. E. bandits in Vacor's spell or 
Warden's necklace in Cleansing Flagstone)

What version of the product are you using? On what operating system?
0.6.7

Please provide any additional information below.

Please either fix the looting-multiple-monsters bug or set quest-item bearers 
to re-spawn; at least if you don't have/haven't had that item.
```

Original issue reported on code.google.com by `fetti...@gmail.com` on 27 Dec 2010 at 10:59

**Comment by GoogleCodeExporter** [comment]
```
Loot from multiple monsters work at least for regular monsters, not sure if 
quest monsters are threated differently.

Could you be more specific about which quest monster and regular monster(s) 
that caused the loot issue ?

Also which settings did you use for attack and loot ?
```

Original comment by `kims...@gmail.com` on 27 Dec 2010 at 11:19

**Comment by GoogleCodeExporter** [comment]
```
I use instant-loot and personally got jipped out of the 4th piece of Vacor's 
spell, someone else was noticing similar issues with the prison warden in the 
flagstone quest chain.

I haven't really cared about regular monster loot so i haven't noticed if the 
drop rates change for multiple/single regular mobs, just got stuck on this 
quest due to the non-drop of the 4th piece of the rift spell : \
```

Original comment by `fetti...@gmail.com` on 28 Dec 2010 at 2:16

**Comment by GoogleCodeExporter** [comment]
```
Are you really sure you found all 4 bandits?
I searched for the 4. very long.

Try to start a new game and search them all. Then load your game and check if 
you really killed all...
```

Original comment by `SamuelPl...@gmail.com` on 28 Dec 2010 at 8:11

**Comment by GoogleCodeExporter** [comment]
```
I found them also, very easy really. the last one was the hardest, to find but 
they are there.  I am now level 22 almost 23 and have everything Armor / 
weapons and am waiting for 6.8 to come out.  I can't wait tierd of killing the 
same mobs over and over LOL. No more quests until next version
```

Original comment by `johnmagu...@gmail.com` on 29 Dec 2010 at 12:46

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. Vacor's thieves are good at hiding. After all, they're 
thieves :)

Quest items are always dropped when the monster is killed. If you choose not to 
pick up the item, it will still be there on the ground for you to pick up. If 
you drop it from your inventory, it will remain on the ground where you dropped 
it.

Unless we can reproduce this, I'm considering we close this issue. Ok?
```

Original comment by `oskar.wi...@gmail.com` on 9 Jan 2011 at 7:39

**Comment by GoogleCodeExporter** [comment]
```
Hm, there might have been a bug here. See issue 56, comment 1. 

I'm marking this issue as "Ready" since the fix in issue 56 should have solved 
this now.

Thank you for reporting it.
```

Original comment by `oskar.wi...@gmail.com` on 13 Jan 2011 at 9:55
- Changed state: **Ready**
- Added labels: **Milestone-0.6.8**

**Comment by GoogleCodeExporter** [comment]
```
i fought luthors ghost at the bottom of the catacombs and was attacked by 
another monster at the same time. At the end of combat no key and I left went 
back into catacombs and still no key.
```

Original comment by `loren.r....@gmail.com` on 14 Jan 2011 at 11:31

**Comment by GoogleCodeExporter** [comment]
```
This will be fixed in version 0.6.8
```

Original comment by `sdeva...@gmail.com` on 10 Feb 2011 at 1:43

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.8 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 24 Feb 2011 at 9:35
- Changed state: **Fixed**
